pkg/yurtappmanager/controller/uniteddeployment/pool.go

```diff
@@ -30,6 +30,7 @@ type Pool struct {
 	Namespace string
 	Spec      PoolSpec
 	Status    PoolStatus
+	Patches   string
```

Comment:
How about move "Patch" to Status, called PatchInfo?

Reply:
done